.github/workflows/rust.yml

```diff
@@ -138,13 +138,21 @@ jobs:
         toolchain: ${{ fromJSON(needs.setup.outputs.toolchains) }}
         flags:
           - --all-features
+          - --no-default-features
     steps:
       - name: Checkout source code
         uses: actions/checkout@v2
 
       - name: Remove default members
         run: sed '/default-members/d' -i ${{ matrix.directory }}/Cargo.toml
 
+      - name: Enable default features for nng
```

Comment:
It seems strange to me that we're doing this instead of explicitly adding the `build-nng` flag?

Reply:
We can't pass `build-nng` without having some global awareness, where the feature flag could be passed. It's an error for example to pass `build-nng` to `stateful`.

Reply:
This would be the better code snippet
```bash
while read -r crate; do
    nng_line=$(grep -n '^nng = ' "$crate/Cargo.toml" | cut -d: -f1)
    sed "${nng_line}s/default-features = false/default-features = true/" -i "$crate/Cargo.toml"
done <<< "$(find . -name Cargo.toml -printf '%h\n')"
```